src/msw/handlers.ts

```diff
@@ -0,0 +1,14 @@
+import * as msw from 'msw';
+import singleUser from './mockData/singleUser.json';
+export const handlers = [
+  // Intercept the "GET /resource" request.
+  msw.http.post('https://leetcode.com/graphql', async (ctx) => {
+    const test = await ctx.request.json();
+    const typed = test as { query: string };
+    if (typed.query.indexOf('getUserProfile') !== -1) {
+      return msw.HttpResponse.json(singleUser);
+    }
+
+    return msw.HttpResponse.json({});
+  }),
+];
```

Comment:
msw returns mock responses by intercepting requests. In this case i'm intercepting the request from the getUserProfile call and returning mockData to verify aganist. I got the mockData by simply console logging the requlst of the request to the leetcode graphql api and then subbing in a different name